testsuite/features/step_definitions/docker_steps.rb

```diff
@@ -45,8 +45,28 @@ def retrieve_build_host_id
   repeat_until_timeout(timeout: timeout.to_i, message: 'image build did not complete') do
     image_details = $api_test.image.get_details(image_id)
     log "Image Details: #{image_details}"
-    break if image_details['buildStatus'] == 'completed' && image_details['inspectStatus'] == 'completed'
```

Comment:
ehmmm but if we remove this second check, then we will break before it is ready. Did I miss something?